templates/eap-xp1-amq-s2i.json

```diff
@@ -0,0 +1,917 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "annotations": {
+            "iconClass": "icon-eap",
+            "tags": "eap,eap-xp,javaee,java,jboss,hidden",
+            "version": "1.0",
+            "openshift.io/display-name": "JBoss EAP XP + AMQ 7",
+            "openshift.io/provider-display-name": "Red Hat, Inc.",
+            "description": "An example JBoss Enterprise Application Platform Expansion Pack application using Red Hat JBoss AMQ. For more information about using this template, see https://github.com/jboss-container-images/jboss-eap-openshift-templates/blob/eap-xp1/README.adoc",
+            "template.openshift.io/long-description": "This template defines resources needed to develop a JBoss Enterprise Application Platform continuous delivery based application, including a build configuration, application deployment configuration, using Red Hat JBoss AMQ and secure communication using passthrough TLS.",
+            "template.openshift.io/documentation-url": "https://access.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/",
+            "template.openshift.io/support-url": "https://access.redhat.com"
+        },
+        "name": "eap-xp1-amq-s2i"
+    },
+    "labels": {
+        "template": "eap-xp1-amq-s2i",
+        "xpaas": "1.0"
+    },
+    "message": "A new JBoss EAP XP and AMQ 7 based application with SSL support has been created in your project. The username/password for accessing the AMQ service is ${MQ_USERNAME}/${MQ_PASSWORD}. Please be sure to create the secrets for \"${JGROUPS_ENCRYPT_SECRET}\" containing the ${JGROUPS_ENCRYPT_KEYSTORE} file used for securing JGroups communications.",
+    "parameters": [
+        {
+            "displayName": "Application Name",
+            "description": "The name for the application.",
+            "name": "APPLICATION_NAME",
+            "value": "eap-xp-app",
+            "required": true
+        },
+        {
+            "displayName": "EAP Image Name",
```

Comment:
Should be "JBoss EAP XP Image Name"

To be exact, we should name it "JBoss EAP XP ImageStream Tag" but I'm not sure if that be more confusing to the user...

Reply:
I've clarified this in the description.